Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -203,6 +203,23 @@ lemma nonneg_iff_isPositive (f : E →L[𝕜] E) : 0 ≤ f ↔ f.IsPositive := b
 
 end PartialOrder
 
+/-- A star projection operator is positive.
+The proof of this will soon be simplified to `IsStarProjection.nonneg` when we
+have `StarOrderedRing (E →L[𝕜] E)`. -/
+theorem isStarProjection_isPositive {p : E →L[𝕜] E}
```

Comment:
This should probably either be called `_root_.IsStarProjection.continuousLinearMapIsPositive` or `isPositive.of_isStarPojection`